scripts/tests/bootstrapper/lotus.dockerfile

```diff
@@ -0,0 +1,53 @@
+# Lotus binaries image, to be used in the local devnet with Forest.
+FROM golang:1.21-bullseye AS lotus-builder
+
+RUN apt-get update && apt-get install -y curl ca-certificates build-essential clang ocl-icd-opencl-dev ocl-icd-libopencl1 jq libhwloc-dev 
+
+WORKDIR /lotus
+
+# Install rust toolchain for rebuilding `filecoin-ffi`
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y --no-modify-path --profile minimal
+
+ENV PATH="/root/.cargo/bin:${PATH}"
+
+ARG LOTUS_VERSION
+RUN git clone --depth 1 --branch ${LOTUS_VERSION} https://github.com/filecoin-project/lotus.git .
+
+# Replace the default bootstrap peers with a pre-defined one. This is needed on this level because the bootstrap peers are compiled into the binary and not configurable at runtime.
+ARG BOOTSTRAPPER
+RUN echo ${BOOTSTRAPPER} > build/bootstrap/calibnet.pi
+
+# Apply a patch to Lotus so that local IPs are not discarded
+COPY patch-lotus.diff .
```

Comment:
We could get rid of this custom dockerfile once https://github.com/filecoin-project/lotus/pull/12289 is merged and released.